apps/vpnmanager/contrib/dokku/scripts/apiClient.mjs

```diff
@@ -0,0 +1,26 @@
+export async function apiClient(endpoint, method = "POST", body = null) {
```

Comment:
`apiClient` (noun) implies a something to help you talk to/connect to a server. However, the code appears to use it as a verb i.e. `result = await apiClient(`